src/zc/lockfile/tests.py

```diff
@@ -94,6 +100,92 @@ def pid_in_lockfile():
     >>> lock.close()
     """
 
+
+def hostname_in_lockfile():
+    r"""
+    # hostname is correctly written into the lock file when it's included in the
+    # lock file content template
+    >>> import zc.lockfile
+    >>> with patch('socket.gethostname', Mock(return_value='myhostname')):
+    ...     lock = zc.lockfile.LockFile("f.lock", content_template='{hostname}')
+    >>> f = open("f.lock")
+    >>> _ = f.seek(1)
```

Comment:
You don't need the `seek()` here as you're calling `strip()` anyway on the next line.